settings.gradle

```diff
@@ -4,59 +4,68 @@ rootProject.name = 'zipkin'
 // Zipkin's mostly common library //
 ////////////////////////////////////
 // Not all projects depend on this!
-include 'zipkin-common',
```

Comment:
That's correct, having different include statements caused problems for `gradle idea`. I think if we move just some modules to the bottom that are not dependencies of other projects, it should be fine.